src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,24 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+import './UserInfo.scss';
```

Comment:
Consider using a more descriptive classname, such as 'user-info', for the imported SCSS file, as this would follow the BEM naming convention.